torchci/pages/api/drci/drci.ts

```diff
@@ -105,17 +106,24 @@ async function forAllPRs(workflowsByPR: Map<number, PRandJobs>, func: CallableFu
   );
 }
 
+async function get_head_branch(repo: string) {
```

Comment:
It does not do anything asynchronously, does it?
```suggestion
function get_head_branch(repo: string) {
```